documentation/docs/steps/jenkinsMaterializeLog.md

```diff
@@ -0,0 +1,16 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequsites
+
+none
+
+## ${docGenParameters}
+
+## ${docGenConfiguration}
+
+## ${docJenkinsPluginDependencies}
+
+## Example
+
```

Comment:
I would like to suggest to add some example(s) here.

As we can see the closure `body` requires a log file name. Why not documenting this?